remove-vowels/meta.json

```diff
@@ -0,0 +1,7 @@
+{
+  "title": "Remove Vowels",
+  "name": "remove-vowels",
+  "difficulty": "hard",
+  "author": "Pranjwal Singh",
+  "category": "loops"
```

Comment:
I believe this would be classified as a list problem (specifically list-3, since its a hard question), even if there are loops required.

Reply:
shit you're right! I changed it now!!